### ISSUE TITLE
Not able to compile example program with CMake [ubuntu 16.04]

### ISSUE THREAD

**Issue by vyi** [PointCloudLibrary/pcl#3239]
I built and installed the master branch on my computer.
Now I am trying to read a pcd file and following [this](http://pointclouds.org/documentation/tutorials/reading_pcd.php) tutorial.

## Your Environment

* Ubuntu 16.04
* GCC
* master-branch built and installed 

## Context
I want to get started with using the pcl library and even the first "hello world" program is not building. Its disappointing

## Expected Behavior

I expect the program to compile without much errors.

## Current Behavior

The CMake build says version of FLANN is not ok. So I have built FLANN on my system too and still it is complaining. 

## Code to Reproduce

Just try compiling the "reading pcd tutorial" code with CMake from [this](http://pointclouds.org/documentation/tutorials/reading_pcd.php) tutoriall

## Error Log 
```
CMake Error at /usr/local/share/pcl-1.9/Modules/FindFLANN.cmake:43 (add_library):
  add_library cannot create imported target "FLANN::FLANN" because another
  target with the same name already exists.
Call Stack (most recent call first):
  /usr/local/share/pcl-1.9/PCLConfig.cmake:247 (find_package)
  /usr/local/share/pcl-1.9/PCLConfig.cmake:302 (find_flann)
  /usr/local/share/pcl-1.9/PCLConfig.cmake:527 (find_external_library)
  CMakeLists.txt:4 (find_package)
...
Looking for bla bla..
Found bla bla..
...
 Found PCL_STEREO: /usr/local/lib/libpcl_stereo.so  
-- Found PCL: pcl_common;pcl_octree;pcl_io;pcl_kdtree;pcl_search;pcl_surface;pcl_sample_consensus;pcl_filters;pcl_features;pcl_tracking;pcl_keypoints;pcl_registration;pcl_ml;pcl_recognition;pcl_segmentation;pcl_stereo;/usr/lib/x86_64-linux-gnu/libboost_system.so;/usr/lib/x86_64-linux-gnu/libboost_filesystem.so;/usr/lib/x86_64-linux-gnu/libboost_date_time.so;/usr/lib/x86_64-linux-gnu/libboost_iostreams.so;/usr/lib/x86_64-linux-gnu/libboost_serialization.so;/usr/lib/x86_64-linux-gnu/libboost_regex.so;optimized;/usr/lib/x86_64-linux-gnu/libqhull.so;debug;/usr/lib/x86_64-linux-gnu/libqhull.so;/usr/lib/libOpenNI.so;FLANN::FLANN (Required is at least version "1.7") 
```

**Comment by SergioRAgostinho** [comment]
You are compiling our master branch. It's where development takes place, so situations like this happen. That's why software has the concept of stable releases. If you are interested in getting things running with less likelihood of issues, I would recommend using pcl 1.9.1.

Now back to the issue. Which version of FLANN are you using?

**Comment by vyi** [comment]
@SergioRAgostinho  Thank you for the response.
I will do `git checkout` to a stable release (any suggestions ? maybe 1.8.2) 
I have FLANN 1.9.1 in my system. I do not have VTK though. Is VTK a must?

**Comment by taketwo** [comment]
> I do not have VTK though. Is VTK a must?

It's needed for visualization and some other secondary functionalities. For the tutorial in question you do not need it.

> a stable release (any suggestions ? maybe 1.7.2)

1.9.1 should be good.

> I have FLANN 1.9.1 in my system.

Did you install it from a tag in their repository, or also from `master`?

**Comment by vyi** [comment]
@taketwo Indeed I installed FLANN from master. 

My build seems to be making progress, no FLANN complaints. Now I am getting VTK related errors when trying to build PCL `1.9.1` 

```
CMake Error at cmake/Modules/FindQVTK.cmake:10 (if):
  if given arguments:

    "VERSION_LESS" "6.0"

  Unknown arguments specified
Call Stack (most recent call first):
  CMakeLists.txt:376 (find_package)
```

**Comment by SergioRAgostinho** [comment]
> Indeed I installed FLANN from master.

I'm surprised this worked. We recently patched our master branch because it was not compiling against FLANN master.

> Now I am getting VTK related errors when trying to build PCL 1.9.1

You need to install QVTK.
```
$ sudo apt install libvtk6-dev libvtk6-qt-dev 
```

**Comment by vyi** [comment]
I installed the two packages `libvtk6-dev` and `libvtk6-qt-dev` but I am getting the same errors.

```
-- VTK_MAJOR_VERSION , rendering backend: OpenGL
CMake Error at cmake/Modules/FindQVTK.cmake:10 (if):
  if given arguments:

    "VERSION_LESS" "6.0"

  Unknown arguments specified
Call Stack (most recent call first):
  CMakeLists.txt:376 (find_package)


-- Configuring incomplete, errors occurred!

```

The package manager shows `libvtk6.2-*` installed in addition to the `libvtk6-*` 
 
![p_man](https://user-images.githubusercontent.com/23264670/61983809-a776cc00-afb6-11e9-8d31-c08b87e4ebd4.png)


To remedy the situation I built `vtk-8.2` from source too and installed it in my system. But still couldn't get rid of those errors. 

The error suggests problems with FindQVTK.cmake script.
